!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  children_per_node_: 8
  goal_z_param: 3.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      children_per_node_: 8
      goal_z_param: 3.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      max_point_age_s_: 20.0
      min_num_points_per_cell_: 1
      min_sensor_range_: 0.2
      n_expanded_nodes_: 40
      name: Default
      obstacle_cost_param_: 8.5
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      pitch_cost_param_: 25.0
      sensor_range_: 15.0
      smoothing_margin_degrees_: 40.0
      smoothing_speed_xy_: 10.0
      smoothing_speed_z_: 3.0
      state: true
      timeout_critical_: 0.5
      timeout_startup_: 5.0
      timeout_termination_: 15.0
      tree_heuristic_weight_: 35.0
      tree_node_duration_: 0.5
      type: ''
      velocity_cost_param_: 6000.0
      yaw_cost_param_: 3.0
    state: []
  max_point_age_s_: 20.0
  min_num_points_per_cell_: 1
  min_sensor_range_: 0.2
  n_expanded_nodes_: 40
  obstacle_cost_param_: 5.0
  pitch_cost_param_: 25.0
  max_sensor_range_: 15.0
  smoothing_margin_degrees_: 40.0
  smoothing_speed_xy_: 10.0
  smoothing_speed_z_: 3.0
  timeout_critical_: 0.5
  timeout_startup_: 5.0
  timeout_termination_: 15.0
  tree_heuristic_weight_: 35.0
  tree_node_duration_: 0.5
  velocity_cost_param_: 6000.0
  yaw_cost_param_: 3.0
state: []
